app/controllers/reactions_controller.rb

```diff
@@ -0,0 +1,33 @@
+class ReactionsController < ApplicationController
+  before_action :require_authentication, only: %i[update destroy]
+  before_action :find_or_initialize_reaction, only: %i[set_from_email update destroy]
+
+  def set_from_email
+    @reaction.update!(reaction_name: params[:reaction_name])
```

Comment:
Just to check, will `.update!` work if it's a new/un-persisted record?

Reply:
Yeah! It does

Reply:
Ah that's good to know 👍